qcfractal/cli/qcfractal_manager.py

```diff
@@ -90,7 +91,7 @@ class DaskQueueSettings(BaseSettings):
 
     def __init__(self, **kwargs):
         """Enforce that the keys we are going to set remain untouched"""
-        forbidden_set = {"name", "ncores", "memory", "processes", "walltime", "env_extra", "qca_resource_string"}
+        forbidden_set = {"name", "cores", "memory", "processes", "walltime", "env_extra", "qca_resource_string"}
```

Comment:
Why this switch? It is `ncores` everywhere else.

Reply:
Its `cores` here because the forbidden set is for *dask specific keywords*, so the user cannot overload the `SLURMCluster(cores=...)` kwarg through extra keys in the model.